Animate/world.ts

```diff
@@ -26,8 +26,9 @@ export const enum Has {
 }
 
 export class World {
-    // Component flags
     Signature: Array<number> = [];
+    Graveyard: Array<number> = [];
```

Comment:
I think it should be more specific:
```suggestion
    Graveyard: Array<Entity> = [];
```